src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -166,6 +171,134 @@ tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
   return deriv_spatial_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+void time_deriv_of_spatial_metric(
+    const gsl::not_null<tnsr::ii<DataType, SpatialDim, Frame>*>
+        dt_spatial_metric,
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept {
+  if (get_size(get<0, 0>(*dt_spatial_metric)) != get_size(get(lapse))) {
+    tnsr::ii<DataType, SpatialDim, Frame> dt_spatial_metric_(get(lapse));
```

Comment:
I'd prefer this be done without the temporary variable, but it's not enough to hold up the PR on its own. lines 183-184 can be
```
*dt_spatial_metric = tnsr::ii<DataType, SpatialDim, Frame>{get(lapse)};
```
I believe that it is also allowable to construct with `get(lapse).size()`, which would avoid a copy.

Reply:
changed